pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,47 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from django.core.management.base import BaseCommand
 
+from django_redis import get_redis_connection
 
-class Command(PootleCommand):
-    help = "Flush stats cache."
+from pootle.core.mixins.treeitem import POOTLE_DIRTY_TREEITEMS
+from pootle.core.models import Revision
+from pootle_store.models import Unit
 
-    def handle_all_stores(self, translation_project, **options):
-        translation_project.clear_all_cache()
+
+class Command(BaseCommand):
```

Comment:
So `flush_cache` would be a command that adds the ability to remove the dirty counts and other RQ data (but no stats)? So `clear_stats` would be left as it was before the PR? If answer is yes in both cases I don't disagree.